packages/react/src/table/docs/overview.mdx

```diff
@@ -513,7 +517,9 @@ Some tables require cells which represent multiple rows or columns. Instead of r
 
 The actions associated with each table row (e.g. ‘Edit’, ‘Delete’, ‘Download’) should always be positioned in the last column, beneath the table header labeled ‘Actions’.
 
-If each table row can open a page to view more information, the cells in the first column should be displayed as bold clickable links under the table header labeled with the unique identifier.
+Dropdown menus can be used to contain three or more essential actions in tables. Labels should be incluided on dropdown menu buttons.
+
+If each table row can open a page or drawer to view more information, the cells in the first column should be displayed as bold clickable links under the table header labeled with the unique identifier.
```

Comment:
I don't think "or drawer" is right here. Drawer's shouldn't be used to view information in a table row like this, only to edit data in the row

Reply:
```suggestion
If each table row can open a page to view more information, the cells in the first column should be displayed as bold clickable links under the table header labelled with the unique identifier.

Links that open a page to view a record or more information should be located in the first column, be labelled with a unique identifier, and display as bold text.
```

Reply:
@adrian-yee Why do we have this second "Links that open…" paragraph? All of that content is listed in the paragraph above

Reply:
my bad, I was proposing the second para as an alternative to the first

Reply:
Oh. Updated